.github/workflows/verify-devcontainer-base.yml

```diff
@@ -0,0 +1,11 @@
+name: Verify make cross in devcontainer base
+on:
+  workflow_dispatch:
+jobs:
+  publish_release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run `make cross` against devcontainer base
+        run: |
+        docker run registry.access.redhat.com/ubi9/go-toolset:latest \
```

Comment:
I'd run `podman build` against `.devcontainer/Containerfile`, and then use the built image with `sh -c "git clone https://github.com/crc-org/crc && cd crc && make cross"`.
`registry.access.redhat.com/ubi9/go-toolset:latest` will be missing some tools required for building crc, or we would not need this intermediate image.

Reply:
the only thing that devcontainer needs is a reinstall of git-core for some reason. this is already part of the actual image, so not sure why this caused issues with this not being found.

I would prefer not to use intermediate containers that gets published. but you mean to just perform a local build `-t devcontainer` and use that for the `docker run`, right?

Reply:
using `ubuntu-24.04`
```
        podman build -t devcontainer -f .devcontainer/Containerfile
        podman run devcontainer \
          sh -c "git clone https://github.com/crc-org/crc --depth 1 && cd crc && make cross"
```